<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureDialog</class>
 <widget class="QDialog" name="CaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture data - Veusz</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Datasets:</string>
       </property>
       <property name="buddy">
        <cstring>descriptorEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="descriptorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Enter a descriptor to describe the format of the incoming data, e.g. &quot;x,+,- y,+-&quot; (see the Data-&gt;Import dialog box for details)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="captureGroup">
     <property name="title">
      <string>Capture method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="captureFileButton">
        <property name="text">
         <string>&amp;File or named pipe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Filename:</string>
          </property>
          <property name="buddy">
           <cstring>filenameEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HistoryCombo" name="filenameEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Browse for file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="captureInternetButton">
        <property name="text">
         <string>Connect to &amp;socket</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="buddy">
           <cstring>hostEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HistoryCombo" name="hostEdit">
          <property name="text" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HistoryCombo" name="portEdit">
          <property name="toolTip">
           <string>TCP port to connect to</string>
          </property>
          <property name="text" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="captureProgramButton">
        <property name="text">
         <string>E&amp;xternal program</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Command line:</string>
          </property>
          <property name="buddy">
           <cstring>commandLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HistoryCombo" name="commandLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stopGroup">
     <property name="title">
      <string>Stop after</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="clickingStopButton">
        <property name="text">
         <string>Clicking fi&amp;nish button</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="HistoryCombo" name="numLinesStopEdit">
        <property name="text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="numLinesStopButton">
        <property name="text">
         <string>Number of input &amp;lines</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="timeStopButton">
        <property name="text">
         <string>Total &amp;time period (s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="HistoryCombo" name="timeStopEdit">
        <property name="text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="HistoryCheck" name="updateIntervalsCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Update document at intervals (s)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="HistoryCombo" name="updateIntervalsEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="HistoryCheck" name="tailCheck">
       <property name="text">
        <string>Only retain latest N values</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="HistoryCombo" name="tailEdit">
       <property name="toolTip">
        <string>Maximum number of values to retain</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryCheck</class>
   <extends>QCheckBox</extends>
   <header>historycheck.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CaptureDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
